ccpp/physics_namelists/input_GFS_v15_FA.nml

```diff
@@ -0,0 +1,139 @@
+&gfs_physics_nml
+       fhzero         = 6.
+       ldiag3d        = .true.
+       ldiag_ugwp     = .false.
+       do_ugwp        = .false.
+       do_tofd        = .false.
+       fhcyc          = 24.
+       nst_anl        = .true.
+       use_ufo        = .true.
+       pre_rad        = .false.
+       ncld           = 5
+       imp_physics    = 15
+       spec_adv       = .true.
+       RHGRD          = 0.98
+       pdfcld         = .false.
+       fhswr          = 3600.
+       fhlwr          = 3600.
+       ialb           = 1
+       iems           = 1
+       IAER           = 111
+       ico2           = 2
+       isubc_sw       = 2
+       isubc_lw       = 2
+       isol           = 2
+       lwhtr          = .true.
+       swhtr          = .true.
+       cnvgwd         = .true.
+       shal_cnv       = .true.
+       cal_pre        = .false.
+       redrag         = .true.
+       dspheat        = .true.
+       hybedmf        = .true.
+       satmedmf       = .false.
+       do_myjsfc      = .false.
+       do_myjpbl      = .false.
+       shinhong       = .false.
+       do_ysu         = .false.
+       lheatstrg      = .false.
+       lgfdlmprad     = .false.
+       effr_in        = .false.
+       random_clds    = .false.
+       trans_trac     = .false.
+       cnvcld         = .true.
+       imfshalcnv     = 2
+       imfdeepcnv     = 2
+       cdmbgwd        = 3.5,0.25
+       prslrd0        = 0.
+       ivegsrc        = 1
+       isot           = 1
+       lsm            = 1
+       iopt_dveg      = 2
+       iopt_crs       = 1
+       iopt_btr       = 1
+       iopt_run       = 1
+       iopt_sfc       = 1
+       iopt_frz       = 1
+       iopt_inf       = 1
+       iopt_rad       = 1
+       iopt_alb       = 2
+       iopt_snf       = 4
+       iopt_tbot      = 2
+       iopt_stc       = 1
+       debug          = .false.
+       oz_phys        = .false.
+       oz_phys_2015   = .true.
+       h2o_phys       = .true.
+       nstf_name      = 2,1,1,0,5
+       xkzminv        = 0.3
+       xkzm_m         = 1.0
+       xkzm_h         = 1.0
+       do_sppt        = .false.
+       do_shum        = .false.
+       do_skeb        = .false.
+       do_sfcperts    = .false.
+/
+
+&gfdl_cloud_microphysics_nml
```

Comment:
Just for curiosity, do we need this section for FA?

Reply:
Not that I know of. This was just a copy/paste thing.